rootfs-builder/rootfs.sh

```diff
@@ -19,6 +19,11 @@ if [ -n "$DEBUG" ] ; then
 	set -x
 fi
 
+if [ -z "$GOPATH" ] ; then
```

Comment:
Hi @harche - thanks for raising this.

Please could you move this check to below the `getopts` call below to allow you to make use of `die` [*], which will send the message to `stderr` with the standard error prefix we use:

```
[ -z "$GOPATH" ] && die "GOPATH not set"
```

---
[*] - we use this Perl-like idiom in lots of scripts by the way.